src/app/3d/qgsmeshlayer3drendererwidget.cpp

```diff
@@ -37,6 +37,7 @@ QgsMeshLayer3DRendererWidget::QgsMeshLayer3DRendererWidget( QgsMeshLayer *layer,
   mWidgetMesh = new QgsMesh3dSymbolWidget( layer, this );
   mWidgetMesh->configureForDataset();
   layout->addWidget( mWidgetMesh );
+  mWidgetMesh->setEnabled( false );
```

Comment:
An alternative could be to call [onEnabledClicked()](https://github.com/qgis/QGIS/blob/master/src/app/3d/qgsmeshlayer3drendererwidget.cpp#L84). Don't know which one is better/safer/less expensive...